.azure-devops/templates/smoke-tests.yml

```diff
@@ -0,0 +1,50 @@
+parameters:
```

Comment:
There are steps which should not happen in the smoke test stage, like:

![image](https://user-images.githubusercontent.com/28658112/153056418-b9a3c932-9896-4ab6-b30d-c704d3ee9817.png)

Reply:
I had to include this to be able to run the c2d command (requires uamqp version update)

Reply:
![image](https://user-images.githubusercontent.com/51140335/153056827-cbae6bc4-3677-4b0a-8b50-76768d230cdb.png)
The script just hangs here if I don't include this step

Reply:
Then pass `-y` to the command to automatically install uamqp.

Reply:
got it